smsyn/specmatch.py

```diff
@@ -31,21 +32,20 @@ def grid_search(spec, libfile, wav_exclude, param_table, idx_coarse, idx_fine):
             search.
     """
     wavlim = spec.wav[0],spec.wav[-1]
-    lib = smsyn.library.read_hdf(libfile,wavlim=wavlim)
+    lib = smsyn.library.read_hdf(libfile, wavlim=wavlim)
     wavmask = wav_exclude_to_wavmask(spec.wav, wav_exclude)
     match = smsyn.match.Match(
         spec, lib, wavmask, cont_method='spline-dd', rot_method='rot'
     )
     
     # First do a coarse grid search
     print "performing coarse grid search"
-    param_table_coarse = grid_search_loop(match, param_table.ix[idx_coarse])
+    param_table_coarse = grid_search_loop(match, param_table.iloc[idx_coarse])
```

Comment:
I think we want `.loc[]`.  `ix[]` was the old way to slice a dataframe based its index. the new way `loc[]`. .iloc[] does the standard numpy array slicing where you slice based on order in the array. Can you check that loc works?

Reply:
Good catch, `.loc` is indeed what we want. Testing now.

Reply:
Testing (on NRES data) complete and results are identical. Ready to merge.